presentation/common/src/presentation-common/PresentationRpcInterface.ts

```diff
@@ -170,7 +170,15 @@ export class PresentationRpcInterface extends RpcInterface {
   /** @beta */
   public async getContentSources(_token: IModelRpcProps, _options: ContentSourcesRpcRequestOptions): PresentationRpcResponse<ContentSourcesRpcResult> { return this.forward(arguments); }
 
-  public async getContentDescriptor(_token: IModelRpcProps, _options: ContentDescriptorRpcRequestOptions): PresentationRpcResponse<DescriptorJSON | undefined> { return this.forward(arguments); }
+  public async getContentDescriptor(_token: IModelRpcProps, _options: ContentDescriptorRpcRequestOptions): PresentationRpcResponse<DescriptorJSON | undefined> {
+    arguments[1].transport = "unparsed-json";
```

Comment:
Why not set it through `options.transport`?

Reply:
iTwin.js RPC implementation expects RPC interfaces to call their `forward` method with received `arguments` object which contain an additional parameter for invoked RPC operation name. Modifying the `arguments` object is the most straightforward way to augment `options` without breaking this assumption.

Because `options` object is modified directly, the operation caller will be able to observe `transport` property being added. I have pushed a fix in 116e0bda3316a8e24294126de7a2c87fd277e1f7 that removes this side-effect.